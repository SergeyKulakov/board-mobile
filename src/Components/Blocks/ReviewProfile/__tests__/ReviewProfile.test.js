import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import Component from '../ReviewProfile'

describe('ReviewProfile', () => {
  const defaultProps = {
    avatar: 'avatar',
    userId: 'userId',
    userName: 'username',
    rate: 3,
    reviewsCount: '4',
    t: text => text,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
