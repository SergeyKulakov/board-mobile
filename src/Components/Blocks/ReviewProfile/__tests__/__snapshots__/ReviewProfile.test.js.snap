// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewProfile render snapshot 1`] = `
<Styled(View)>
  <Styled(View)>
    <Styled(ProviderPhoto)
      avatarURL="avatar"
      username="userId"
    />
  </Styled(View)>
  <Styled(Text)>
    username
  </Styled(Text)>
  <Styled(View)>
    <Styled(Styled(View))>
      <Component
        value={3}
      />
    </Styled(Styled(View))>
    <Styled(Styled(View))>
      <Styled(Text)>
        4
         
        jobDetail.reviews
      </Styled(Text)>
    </Styled(Styled(View))>
  </Styled(View)>
</Styled(View)>
`;
